recipes/pycondor/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "PyCondor" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "182d4bf0e35740c7232f00f3a3259369045514666f14ed4c2bd5751940ec1b9c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like there is a python entry point. Please add it:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and a corresponding test:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands